### DISCUSSION TITLE
FIP Editors RetroPGF Rewards

### DISCUSSION THREAD

**Discussion by jsoares** [filecoin-project/FIPs#1078] [General]
The FIP Editors group was created by FIP-0001, which tasks Editors with ensuring that FIPs are sound and complete, and with stewarding their progress along the pipeline. In line with their responsibilities, FIP Editors have provided thousands of review comments and suggestions on FIP PRs, tended to the pipeline, and contributed to governance discussions, while remaining committed to neutrality and helping improve and advance proposals irrespective of their authorship.

The group touched every one of the 90 FIPs in Filecoin's history. In addition to its core governance responsibilities, outlined in FIP-0001, Editors have always been highly involved in governance discussions and in educating and supporting contributors to the protocol.

Despite some of the FIP Editors being employed by PL and FF, none are employed as FIP Editors and all pursue this role as volunteer public good work, in addition to their primary responsibilities. This work was not rewarded until recently -- then Filecoin RetroPGF happened.

I'm creating this thread to document the rewards received by FIP Editors in the scope of RPGF, both for the sake of transparency -- important given the governance responsibilities of this role -- and in the hope that it will draw more volunteers -- this is still less lucrative than _any other thing_ one could be doing, but the fact that the community values the work and chooses to allocate funding thereto is worth much more than the $ amount. Do consider becoming a FIP editor and sharing in the ❤️ 

I'll create threads below for the past and present rounds. Stay tuned.

**Comment by jsoares** [comment]
#### FIL-RetroPGF-1 (2024Q2)

**Nomination**: https://github.com/filecoin-project/community/discussions/694#discussioncomment-8865946
**Application**: https://www.fil-retropgf.io/projects/0x173ad12dc7dde47816b620ac33b910e8be0619e2120d931e414bee91503a7ce8
**Allocation**: 3442 FIL (1.75%)

Editors active during eligibility period: `@anorth, @arajasek, @jennijuju, @jsoares, @kaitlin-beegle, @momack2, @raulk, @TippyFlitsUK`

We received votes from 9 badge holders and placed 13th out of 106 eligible applications. The 3442 FIL allocation was split into equal parts among the 8 editors who were active at any point in the round's "impact period". Editors were free to assign their part to another entity, and 5 chose to donate it to the FilOz team 🫶 

You'll find more details on the process and results [here](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Results-9ad30232b643460598353489d79b9628).

**Comment by jsoares** [comment]
#### FIL-RetroPGF-2 (2024Q4)

**Nomination**: https://github.com/filecoin-project/community/discussions/714#discussioncomment-10919631
**Application**: https://www.fil-retropgf.io/projects/0xbf6dce4265897f40834378cf9b95dd96eef0d9025f63ac8e4bc2850fa958be1d
**Allocation**: 2126 FIL

Editors active during eligibility period: `@anorth, @jennijuju, @jsoares, @kaitlin-beegle, @momack2, @raulk, @TippyFlitsUK, @rvagg`

We received votes from 14 badge holders and placed 44th out of 153 eligible applications. Fund assignment and distribution still in progress.

Round details and results [here](https://fil-retropgf.notion.site/Round-2-Results-160d0d646da180b09e0ee8baa3c1dfb9).

**Comment by Fatman13** [comment]
I thought discussion is reserved for anything that is subject to community governance. This kind one-way announcement seems more suitable to live in README.md or other documentations?

**Reply by jsoares** [reply]
I'm not sure where that idea comes from. There are several categories in this discussion section, including one called "Announcements". In particular, the [welcome post](https://github.com/filecoin-project/FIPs/discussions/194) reads:

> The only guideline is that this forum be limited to FIP -relevant conversations

Moreover, check out the definition of the "General" topic:

> General
> A space for Web3 and governance conversation topics, etc.

I would argue that the staffing and rewarding of FIP Editors meets both guidelines.

Lastly, this isn't an announcement! The topic is not locked and the whole point of sharing this information is to foster discussion around the editor role and encourage more participation.

**Reply by Fatman13** [reply]
> The only guideline is that this forum be limited to FIP -relevant conversations

The next half of the sentence literally says...

> i.e., ideas for changing something about how Filecoin works.

Bookkeeping how much retroPGF rewards are allocated to editors is not "ideas for changing something about how Filecoin works", imho. And later...

> The FIP Discussion Forum is a place for ideas to live until they are ready to become FIP proposals. Once a FIP proposal is ready to be drafted, authors are asked to do so by opening up an 'Issue' in the FIPs repo.

There is no proposal but bookkeepings AFAIU the op.

**Reply by jsoares** [reply]
We seem to be interpreting the same text very differently, so I'll defer to @kaitlin-beegle as the author. No objections on my end if she wants to move the discussion.

**Reply by Fatman13** [reply]
> We seem to be interpreting the same text very differently

Trying my best with my poor command of the English language. 

imho, i think it's fairly easy to draw the line if the op is a proposal (for discussion) or just bookkeeping. If op is indeed a proposal and subject to community governance, instead of "allocation was split into equal parts among the 8 editors" i would counter propose the following...

- Reward editors based on meritocracy for long term sustainability 
- A clear defined metrics to measure performance of each editor
- A community vote to determine how much each editor get and voter pool should not be drawn from organizations where editors are from

But since rewards have already been allocated to individuals AFAIU, then op to me seems to be just bookkeepings which are not subject to community governance and thus doesn't belong to FIP discussions.

**Comment by 15012700225** [comment]
This is really a shameful moment. In Filecoin's half-dead state, only FIP editors dare not say that they have no responsibility, but the irony is that they will be rewarded.

**Comment by alberto-gonzales** [comment]
@jsoares 

> I'm creating this thread to document the rewards received by FIP Editors in the scope of RPGF, both for the sake of transparency -- important given the governance responsibilities of this role -- and in the hope that it will draw more volunteers -- this is still less lucrative than any other thing one could be doing, but the fact that the community values the work and chooses to allocate funding thereto is worth much more than the $ amount. Do consider becoming a FIP editor and sharing in the ❤️

considered. How?

**Reply by jsoares** [reply]
Glad to hear you're interested! There is no hard process, but my recommendation is to join #fip-fips on slack and start showing your interest and understanding of the protocol, by voluntarily reviewing FIP PRs and participating in the broader process. 

See the latest FIP Editor nomination [here](https://github.com/filecoin-project/FIPs/pull/1053#issue-2501565363). At the point of nomination, Rod had already [reviewed a number of FIP PRs](https://github.com/filecoin-project/FIPs/pulls?q=is%3Apr+reviewed-by%3Arvagg+is%3Aclosed+merged%3A%3C2024-09-02) despite not being an editor. That's really the most straightforward way, and your contributions will already be making a difference!

**Reply by alberto-gonzales** [reply]
that sounds like straight a route forward. thanks @jsoares
